triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -118,6 +98,23 @@ object Machine extends StrictLogging {
 
 object Trigger extends StrictLogging {
 
+  private[trigger] def newLoggingContext[P, T](
+      triggerDefinition: Identifier,
+      triggerActAs: Party,
+      triggerReadAs: Set[Party],
+      extraLoggingEntries: LoggingEntry*
+  ): (LoggingContextOf[Trigger with P] => T) => T = {
+    val args =
+      ("triggerDefinition" ->
+        LoggingValue.OfString(triggerDefinition.toString)) +:
+        ("triggerActAs" ->
+          LoggingValue.OfString(Tag.unwrap(triggerActAs))) +:
+        ("triggerReadAs" ->
+          LoggingValue.OfString(Tag.unsubst(triggerReadAs).mkString("[", ", ", "]"))) +:
```

Comment:
I think you can use `OfIterable` instead or use the `ToLoggingValue` implicit. We should really be consistent in how we print that stuff.

Reply:
dropped.

Reply:
I let the implicit magic does its work.